src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer.Xaml/XamlLspWorkspaceManagerFactory.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Composition;
+using Microsoft.CodeAnalysis.Host.Mef;
+
+namespace Microsoft.CodeAnalysis.LanguageServer.Xaml;
+
+[ExportLspServiceFactory(typeof(LspWorkspaceManager), StringConstants.XamlLspLanguagesContract), Shared]
+internal class XamlLspWorkspaceManagerFactory : LspWorkspaceManagerFactory
+{
+    [ImportingConstructor]
+    [Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
+    public XamlLspWorkspaceManagerFactory(LspWorkspaceRegistrationService lspWorkspaceRegistrationService) : base(lspWorkspaceRegistrationService)
```

Comment:
same here, this should not be necessary when we support xaml requests from the main ROslyn server.

Reply:
You're right, I just verified it's not needed at all.

Reply:
Gone